features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx

```diff
@@ -67,45 +68,94 @@ const useStyles = makeStyles(theme => ({
  * @class SampleAPI
  * @extends {Component}
  */
-function SampleAPI(props) {
+function SampleAPI(prop) {
     const classes = useStyles();
+    const { intl } = prop;
 
     const [step, setStep] = useState(0);
     const [productPath, setProductPath] = useState(null);
+    let calculatorApiId = null;
+    let mathApiId = null;
+
+    /**
+     * Create API Product
+     * @param {*} calculatorApiId
+     * @param {*} mathApiId
+     */
+    function createSampleProduct(calId, mathId) {
+        setStep(3);
+        const sampleProductPayload = apiProduct(calId, mathId);
+        const productRestApi = new APIProduct();
+
+        const productPromise = productRestApi.create(sampleProductPayload);
+        productPromise
+            .then((prod) => {
+                setStep(4);
+                setProductPath(`/api-products/${prod.body.id}/overview`);
+                Alert.info(intl.formatMessage({
+                    id: 'Apis.Listing.SampleAPI.SampleAPIProduct.successful',
+                    defaultMessage: 'CalculatorAPIProduct published successfully',
```

Comment:
How about saying Sample API Product published successfully.

Reply:
Modified it to 'Sample CalculatorAPIProduct' as  we have made the sample api message as  'Sample Pizzashack API'

Reply:
Yes I think it's better to say 

```suggestion
                    defaultMessage: 'Sample API Product published successfully',
```

Reply:
Don't we need to add the name of the APIProduct?